core/lib/libmbedtls/src/tee_lmd_provider.c

```diff
@@ -1263,18 +1294,41 @@ TEE_Result crypto_mac_init(void *ctx, uint32_t algo, const uint8_t *key,
 #endif
 #if defined(CFG_CRYPTO_CMAC)
 	case TEE_ALG_AES_CMAC:
-		return TEE_ERROR_NOT_SUPPORTED;
+		cipher_info =
+			tee_algo_to_mbedtls_cipher_info(TEE_ALG_AES_ECB_NOPAD,
```

Comment:
The name of this function is obviously a bit long, since it's static how about `get_cipher_info()` instead?

Reply:
Yes, it is really a bit long. Your suggestion is good for me.